### ISSUE TITLE
bot references a non-existing template

### ISSUE THREAD

**Issue by mariolenz** [ansible-community/collection_bot#29]
> Please set the description of this issue with this template:
> https://raw.githubusercontent.com/ansible/ansible/devel/.github/ISSUE_TEMPLATE/bug_report.md

[source](https://github.com/ansible-collections/community.vmware/issues/880#issuecomment-856274815)

But https://raw.githubusercontent.com/ansible/ansible/devel/.github/ISSUE_TEMPLATE/bug_report.md doesn't exist.

**Comment by mariolenz** [comment]
I think this has already been fixed in ansible/ansibullbot#1322 but I don't know how to "backport" this. Should I just copy&paste?

**Comment by dmsimard** [comment]
Hi @mariolenz, good catch and thanks for the issue !

We're in fact in the process of re-basing this out-of-date fork of ansibullbot and upstreaming what we can in terms of improvements to handle collections, you can find a WIP here: https://github.com/ansible-community/collection_bot/pull/28

I would expect the issue to be fixed when running from that branch since it contains the fix you refer to but we're not quite ready to make the switch just yet.

In the meantime, I could do a test by creating an issue with the same content and run the bot off of the branch to see what happens.

@jillr: FYI ^

**Comment by dmsimard** [comment]
I did a test by creating this issue: https://github.com/ansible-collections/community.vmware/issues/895

It looks like it works OK.

**Comment by mariolenz** [comment]
resolved_by #28

**Comment by mariolenz** [comment]
@dmsimard Yes, this looks better. I don't get a `404: Not Found` any more.

> We're in fact in the process of re-basing this out-of-date fork of ansibullbot and upstreaming what we can in terms of improvements to handle collections

A bit off-topic, maybe I should open a new issue for this... but when someone opens an issue I often miss the information what version of the collection is used. The one that comes with ansible or a different version installed via galaxy? Maybe this question should be part of the template.

**Comment by dmsimard** [comment]
> @dmsimard Yes, this looks better. I don't get a `404: Not Found` any more.

\o/

> A bit off-topic, maybe I should open a new issue for this... but when someone opens an issue I often miss the information what version of the collection is used. The one that comes with ansible or a different version installed via galaxy? Maybe this question should be part of the template.

My understanding is that the issue template for community.vmware is provided by the default template for the ``ansible-collections`` organization at https://github.com/ansible-collections/.github/tree/main/.github/ISSUE_TEMPLATE.

If you'd like to override the default for a specific repository, you can create the ``.github/ISSUE_TEMPLATE/bug_report.md`` file in the repository and it will have precedence over the organization one.

That said, I think it would be relevant for every collection in the namespace to add a question around what version of a collection is installed and how it was installed. Would you like to send a pull request ? Although since it impacts every collection we might want to get it approved during this week's #ansible-community meeting before merging the change.

**Comment by mariolenz** [comment]
> Although since it impacts every collection we might want to get it approved during this week's #ansible-community meeting before merging the change.

As you say, this impacts every collection. So I don't want to send a pull request just like this. I've opened issue #30 to discuss this and asked on #ansible-community for people to comment.
